src/main/java/io/github/syst3ms/skriptparser/parsing/ScriptLoader.java

```diff
@@ -114,7 +118,11 @@
                     if (items.size() == 0 ||
                             !(items.get(items.size() - 1) instanceof Conditional) ||
                             ((Conditional) items.get(items.size() - 1)).getMode() == Conditional.ConditionalMode.ELSE) {
-                        logger.error("An 'else if' must be placed after an 'if'", ErrorType.STRUCTURE_ERROR);
+                        logger.error(
+                                "An 'else if' must be placed after an 'if'",
+                                ErrorType.STRUCTURE_ERROR,
+                                "Make sure the previous section you used was an 'if'-statement. If not, convert that section to an 'if'-statement or replace this line with one"
```

Comment:
This wasn't clear in the original error message, but an `else if` can also follow another `else if`.

Reply:
Yeah indeed. I’ll state it in the original error message and change the tip accordingly.